packages/ui-extensions/src/surfaces/admin/api/customer-segmentation-template/customer-segmentation-template.ts

```diff
@@ -18,11 +18,27 @@ type TemplateCategory =
   | 'purchaseBehaviour'
   | 'location';
 
+type UnitSystem = 'IMPERIAL_SYSTEM' | 'METRIC_SYSTEM';
+
+interface Location {
+  name: string;
+  latitudeDecimalDegrees?: number;
+  longitudeDecimalDegrees?: number;
+}
+
+interface Shop {
+  b2bEnabled: boolean;
```

Comment:
I see little difference between EnabledFeatures and Shop object. Using either or is ok to me. I see the experiment, tags, aggregate, still as shop-wide beta/feature. I understand that `b2bEnabled` is a ShopFeature and the other are betas or things that have meaning only in the context of segmentation. If we have to maintain this interface for a year as well, the experiment are not long-lived enough for this either, no?

Reply:
yea, i also feel that they're blurry because `enabledFeatures` and this shop object both imply shop-level features. in the end, it doesn't rly matter which one, so i'm good with this. but also don't see a huge difference from `enabledFeatures` 🤔

Reply:
`enabledFeatures` should eventually disappear IMO. It's gonna be hard to maintain this simple array API when Admin has to support API contracts for 12 months after a deprecation. The way we currently use this attribute is to flag what's currently being rolled out in term of **QL features** (aggregates, products purchased on tags). Once this is done, we should remove it and use CDP beta flags to test new QL features.